App/PHP_HTML/login sample/sign_in_submission.php

```diff
@@ -33,7 +33,7 @@
             <ul class="nav navbar-nav navbar-right">
                 <li><a href="#">Dashboard</a></li>
                 <li><a href="#">Settings</a></li>
-                <li><a href="profile.php">Profile</a></li>
+                <li><a href="#">Profile</a></li>
```

Comment:
Could you explain this.

Reply:
We had an old defective profile.php file in login sample so clicking on the profile link after having signed in would direct to that page and not the correct one.